### PR TITLE
add a separator option to the dim dropdown

### PR DESCRIPTION
**Pull request by kyleshay**
Screen Shot 2020-10-01 at 12 30 06 PM

### CODE REVIEW

File: src/app/search/MainSearchBarActions.tsx [hunk 1]
```diff
@@ -107,21 +107,21 @@ function MainSearchBarActions({
...
+
+    if (isTagValue(selectedTag)) {
+      dispatch(bulkTagItems(tagItems, selectedTag));
     }
   });
 
+  const bulkLock = loadingTracker.trackPromise(async (selectedTag: TagValue) => {
+    // Bulk locking/unlocking
+    const state = selectedTag === 'lock';
+    const lockables = filteredItems.filter((i) => i.lockable);
```
Comment (kyleshay) [inline src/app/search/MainSearchBarActions.tsx:121]: I wonder if we should also filter on the is already locked/unlocked so we can skip some requests. Just would need to hope that DIM matches approximate game state.

File: src/app/dim-ui/Dropdown.m.scss [hunk 2]
```diff
@@ -55,3 +55,8 @@
   padding: 4px 8px;
   color: #999;
 }
+
+.seperator {
```
Comment (bhollis) [inline src/app/dim-ui/Dropdown.m.scss:59]: "separator"

File: src/app/dim-ui/Dropdown.tsx [hunk 3]
```diff
@@ -75,24 +85,30 @@ export default function Dropdown({
...
-                  : (e: any) => {
-                      e.nativeEvent.preventDownshiftDefault = true;
-                    },
-              })}
-            >
-              {item.content}
-            </div>
+            <>
+              {!isOption(item) ? (
+                <div className={styles.seperator} />
```
Comment (bhollis) [inline src/app/dim-ui/Dropdown.tsx:90]: This needs a key

File: src/app/dim-ui/Dropdown.m.scss [hunk 4]
```diff
@@ -55,3 +55,8 @@
...
+  height: 1px;
+  background-color: #999;
```
Comment (bhollis) [inline src/app/dim-ui/Dropdown.m.scss:61]: It's a bit bright, can we tone it down?

Comment (kyleshay) [reply src/app/dim-ui/Dropdown.m.scss]: Screen Shot 2020-10-01 at 12 30 06 PM

Comment (kyleshay) [reply src/app/dim-ui/Dropdown.m.scss]: Screen Shot 2020-10-01 at 1 29 34 PM

**Review by bhollis — APPROVED** [review]

**Pull request merged**